CONTRIBUTING.md

```diff
@@ -12,16 +12,33 @@ file.
 
 ## Development prep
 
-- `git clone https://github.com/flutter/devtools`
-- `cd devtools/packages/devtools_app`
-- `flutter pub upgrade`
+1. If you haven't already, follow the [instructions](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) to generate a new SSH key and connect to Github with SSH
+2. Click the "Fork" button to fork `https://github.com/flutter/devtools` into your own Github account
```

Comment:
Instead of writing all these fork instructions out (2-6), you could link to https://docs.github.com/en/get-started/quickstart/fork-a-repo like you did with the SSH key instructions.

Reply:
Good point, updated to use that link instead